tests/rptest/services/compatibility/kafka_streams_examples.py

```diff
@@ -55,8 +55,22 @@ def __init__(self, redpanda, is_driver):
         super(KafkaStreamsWikipedia, self).__init__(redpanda, is_driver,
                                                     jar_args)
 
+        self.users = [
+            "erica", "bob", "joe", "damian", "tania", "phil", "sam", "lauren",
+            "joseph"
+        ]
+
+    def has_user(self, line):
+        found = False
+        i = 0
+        while not found and i < len(self.users):
```

Comment:
This loop could be simplified:
```
for u in self.users:
   if u in line:
       return True
# Fall through
return False
```

Or for best efficiency, you could avoid the loop entirely and search for all usernames in a single path with a regex (depending on how many lines we're searching this may or may not be a noticeable difference):

```
# Precompile the regex and store as a constant earlier, maybe as a class variable on KafkaStreamsWikipedia
users_re = re.compile("(erica|bob|joe)") # ..and the rest of the users
...snip....
def has_user(self, line):
   return self.users_re.users_re.search(line) is not None
```

Reply:
Will do.